cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,20 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
```

Comment:
Your assumption is correct. Once the `split_test` is found in the course tree, its vertical children are visited to fetch the group IDs. IDs and location are used in tandem to update `groups_usage_info`.